src/client/components/editor/info-panel.js

```diff
@@ -0,0 +1,34 @@
+import h from 'react-hyperscript';
+import { Component } from 'react';
+import { DOI_LINK_BASE_URL, PUBMED_LINK_BASE_URL, GOOGLE_SCHOLAR_BASE_URL } from '../../../config';
+
+export class InfoPanel extends Component {
+  constructor(props){
+    super(props);
+  }
+
+  render(){
+    const { document } = this.props;
+
+    if( document.editable() ){ return null; }
+
+    const citation = document.citation();
+    const { authors, pmid, title = 'Untitled article', reference, doi, abstract } = citation;
+
```

Comment:
Since 'anything goes' with regards to user input, we could have  `pmid`, `doi` be `null` and `authors` empty, so these links wouldn't be informative.

Reply:
The content is just filler for demonstration.  We'll have to consider the content of the panel in more detail in future PRs.  This PR is just structural.